src/sentry/api/helpers/span_analysis.py

```diff
@@ -0,0 +1,89 @@
+# example data = list of dictionaries
+
+# data = [{'period': 1, 'k': 'browser', 'txn_count': 2978, 'sum_v': 2088325.678, 'count_v': 24069, 'avg_v': 86.76412306},
+# {'period': 1, 'k': 'http.client', 'txn_count': 2978, 'sum_v': 13444332.97, 'count_v': 78114, 'avg_v': 172.1116953},
+# {'period': 1, 'k': 'mark', 'txn_count': 2978, 'sum_v': 0, 'count_v': 5390, 'avg_v': 0},
+# {'period': 1, 'k': 'pageload', 'txn_count': 2978, 'sum_v': 1853836.58, 'count_v': 2978, 'avg_v': 622.5106045},
+# {'period': 1, 'k': 'paint', 'txn_count': 2978, 'sum_v': 0, 'count_v': 5151, 'avg_v': 0},
+# {'period': 1, 'k': 'resource.css', 'txn_count': 2978, 'sum_v': 449854.7821, 'count_v': 8926, 'avg_v': 50.39825029}]
+
+import collections
+
+
+def span_analysis(data):
+
+    # create a unique identifier for each span
+    span_groups = [row["span_op"] + "," + row["span_group"] for row in data]
```

Comment:
I think this might be a bit more clear because there's already a `span_group` value

```suggestion
    span_keys = [row["span_op"] + "," + row["span_group"] for row in data]
```